### ISSUE TITLE
New suite type -- parallel

### ISSUE THREAD

**Issue by kostja** [tarantool/test-run#6]
In parallel suite it's impossible to manage the server, it is started and stopped between each test.
Each test consists of:
- test config 
- .lua file for prologue
- .lua filies with commands for parallel execution
- .lua file for epilogue

The runner fires off the server, and executes the prologue. If the output is correct, it executes the parallel part in line with settings of test config: number of threads, number of iterations.

At this stage the task of the server is to not crash and not leak memory (it will eventually crash if it leaks memory).

Then the epilogue is executed. The output of the epilogue is compared with the canonical pre-recorded output.

If all steps of the execution passed, then the test is considered passed.

**Comment by rtsisyk** [comment]
Superseded by #7
